apps/omg/test/omg/state/core_test.exs

```diff
@@ -897,6 +898,133 @@ defmodule OMG.State.CoreTest do
            )
   end
 
+  describe "Automatic fees claiming" do
+    setup do
```

Comment:
missing behaviors to test:
 - surpluses adding up for same-token-fees paid in a block
 - surpluses for > 1 token collected, can do 2 fee transactions after that
 - big block tests (see other comment)?
 - surpluses from multi-token outputs are all collected (alternative phrasing: all fees paid from multi-input/output transactions are claimable, so have 4in/4out and each use a different token? A bit of hassle, but I think this is required)
 - if surplus is zero (we have 2ins/2outs tx, both in fee token possibly, only one in/out pay the fee), it won't be collected _at all_ (checks the `Enum.filter(amount > 0)` out there somewhere)